tests/play-with-vars.yml

```diff
@@ -0,0 +1,36 @@
+name: play-with-vars
+vars:
+  aString: '{"foo": "bar"}'
+  multipleVars: {"coo": "car"}
+  multipleVarsSecondSyntax: 
+    goo: gar
+  aQuote: 'foo"bar'
+
+testcases:
+- name: foobar
+  steps:
+  - script: echo '{{.aString}}'
+    info: value of aString is {{.aString}}
+    assertions:
+    - result.systemoutjson.foo ShouldContainSubstring bar
```

Comment:
```suggestion
    - result.systemoutjson.foo ShouldEqual {"foo": "bar"}
```

Reply:
it's result.systemoutjson.foo and not result.systemoutjson used.
replaced by ShouldEqual